pkgs/auto-fix-vscode-server.nix

```diff
@@ -32,6 +32,13 @@ let
       ;
     }) ++ extraFHSPackages pkgs;
 
+    extraBuildCommands = ''
+      if [[ -d /usr/lib/wsl ]]
+      then
+        cp -rsHf /usr/lib/wsl usr/lib/wsl
```

Comment:
Can we symlink?

Reply:
`cp -rsHf` is already symlinking.

Reply:
See https://linux.die.net/man/1/cp#:~:text=%2Dn%20option)-,%2DH,-follow%20command%2Dline

Reply:
It seems `cp -rsHf` is used in nixpkgs to link other libraries.

https://github.com/NixOS/nixpkgs/blob/3d53723112c6045bd6534ebeb536056fec5a16f8/pkgs/build-support/build-fhs-userenv/env.nix#L192

Reply:
@Atry Thanks for clarifying, I wasn't aware it was symlinking either. I will keep it as it is then in PR #47, but I will add a comment about it actually symlinking for clarify sake.